src/js/components/Calendar/__tests__/Calendar-test.tsx

```diff
@@ -102,6 +102,18 @@ describe('Calendar', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('prop header size', () => {
+    const { container } = render(
+      <Grommet>
+        <Calendar size="small" level={3} date={DATE} />
+        <Calendar size="medium" level={2} date={DATE} />
+        <Calendar size="large" level={1} date={DATE} />
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
The snapshots for calendar are really big, I'm wondering if instead of using `toMatchSnapshot` we can use `getByRole` to grab the heading element and then check it's level https://testing-library.com/docs/queries/byrole/#api.

Reply:
agree this is better

Reply:
done